package.json

```diff
@@ -44,6 +44,8 @@
     "@babel/preset-flow": "^7.9.0",
     "cross-env": "^7.0.2",
     "eslint": "^7.0.0",
+    "eslint-5": "npm:eslint@5",
```

Comment:
This is also not necessary. Make the dev dep line identical to the peer dep one, and explicitly install whichever version of eslint you want to test in CI.

Reply:
What are the tradeoffs between these two approaches?

Reply:
It’s less complicated, and works on npm versions that don’t understand the new and rarely used `npm:` protocol.

Also, having 3 eslints installed at once won’t actually be replicating what the user’s node_modules will be like, so the tests are less useful.